python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -104,12 +113,19 @@ def preprocess_ondisk_dataset(
         # Construct the data dict.
         data_dict = {}
         for edge_info in input_config["graph"]["edges"]:
-            edge_data = pd.read_csv(
-                os.path.join(dataset_dir, edge_info["path"]),
-                names=["src", "dst"],
-            )
-            src = torch.tensor(edge_data["src"])
-            dst = torch.tensor(edge_data["dst"])
+            if edge_info["format"] == "numpy":
```

Comment:
is it possible to remove the code duplication here which is same as in homograph code path?